frontend/tsconfig.json

```
@@ -24,6 +24,6 @@
   "angularCompilerOptions": {
     "strictInjectionParameters": true,
     "strictInputAccessModifiers": true,
-    "strictTemplates": true
+    "strictTemplates": true,
```

This is considered invalid JSON by most parsers. Even though TypeScript might read it fine I'd remove it.
```suggestion
    "strictTemplates": true
```